orttraining/orttraining/python/training/orttrainer_options.py

```diff
@@ -435,6 +469,30 @@ def _check_is_callable(field, value, error):
             }
         }
     },
+    'recompute': {
```

Comment:
Is this the most intuitive category? SHould it be under a generic "graph" category, maybe?

Reply:
good idea.